Edge10.CouchDb.Client.Tests/TestChangesParameters.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Edge10.CouchDb.Client.Changes;
```

Comment:
Shouldn't this be under `/tests/changes/TestChangesParameters.cs`  instead of in the root?